.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+closes #???
+
+The following tasks have been completed:
+
+ * [ ] Confirmed there are no ReSpec errors/warnings.
+ * [ ] Modified Web platform tests (link)
+ * [ ] Modified MDN Docs (link)
+ * [ ] Has undergone security/privacy review (link)
```

Comment:
Charter commitment is latest during horizontal review https://www.w3.org/2019/03/devices-sensors-wg-charter.html#coordination

Reviews early and often are encouraged subject to reviewers’ bandwidth.

Reply:
Agree... depends if you have someone who can do the privacy review in the group.